packages/ethereum-contracts/contracts/CID.sol

```diff
@@ -0,0 +1,55 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
where's this and the Varint library from? i can see references to protobuf/multiformats encoders around github, and this seems to reuse parts of those.

Reply:
I've deleted the `Varint.sol` library because we can just import the library from here https://github.com/celestiaorg/protobuf3-solidity-lib (I have updated the CID implementation to just import from this library).

The CID code was written by me, I wrote it by translating the CID implementation in js-multiformats (https://github.com/multiformats/js-multiformats/blob/master/src/cid.ts#L447) into solidity.